### PR TITLE
got

### PR DESCRIPTION
**Pull request by targos**
Fixes: https://github.com/cheminfo/rest-on-couch/issues/185

### CODE REVIEW

**Comment by targos** [comment]
I'm investigating a bug with the streaming version of getAttachment

**Comment by targos** [comment]
btw this is deployed on C6H6

File: src/server/middleware/util.js [hunk 1]
```diff
@@ -83,7 +86,7 @@ async function errorMiddleware(ctx, next) {
 }
 
 function handleCouchError(ctx, e, secure) {
-  if (e.scope !== 'couch') {
+  if (e.name !== 'HTTPError') {
```
Comment (stropitek) [inline src/server/middleware/util.js]: does this result in the exact same behavior as with the nano library?

File: src/util/nanoShim.js [hunk 2]
```diff
@@ -0,0 +1,332 @@
...
+const CouchError = require('./CouchError');
+
+const hasOwnProperty = Object.prototype.hasOwnProperty;
+
+class NanoShim {
+  constructor(url, cookie) {
+    const agent = new http.Agent({
+      timeout: 1000 * 60 * 5
+    });
+    this.client = got.extend({
```
Comment (stropitek) [inline src/util/nanoShim.js]: Nice! That looks a lot like axios.

File: src/server/middleware/util.js (hunk 1 above)
Comment (targos) [reply src/server/middleware/util.js]: I'm not sure. Don't really know how to verify

**Comment by stropitek** [comment]
Close #140

**Comment by stropitek** [comment]
We should merge this after #190

**Pull request merged**